Assets/Scripts/Models/Buildable/Utility.cs

```diff
@@ -80,6 +84,7 @@ private Utility(Utility other)
         typeTags = new HashSet<string>(other.typeTags);
         description = other.description;
         Tint = other.Tint;
+        deconstructInventory = other.deconstructInventory;
```

Comment:
Sure. It makes sense. We will need to copy if we later decide to change the deconstruct inventory for an installed furniture. For example, if a furniture can get damage, maybe it will drop less stuff, or even some other type of inventory, some damaged thing, or something. But we can change it later, if we go in that direction.